### DISCUSSION TITLE
Tuning pressure advance?

### DISCUSSION THREAD

**Discussion by dasb00ter** [bassamanator/Sovol-SV06-firmware#166] [General]
I am trying to use this website [https://ellis3dp.com/Pressure_Linear_Advance_Tool/lines_method_deprecated/](url) to tune an sv06 plus that was just klipperized with this github.  I am confused about what some of the settings should be. I have attempted to put the correct start gcode? Prusa slicer shows retraction length as NA (retraction speed is 25 mms) I am not sure what to put in alot of these windows. Can somebody help me out?
![Screenshot (36)](https://github.com/bassamanator/Sovol-SV06-firmware/assets/36972116/d494b801-9a14-4404-8224-b035e78bae59)
![Screenshot (37)](https://github.com/bassamanator/Sovol-SV06-firmware/assets/36972116/9a26b0d9-5a70-40ed-b5f4-0172eb6a7020)
![Screenshot (38)](https://github.com/bassamanator/Sovol-SV06-firmware/assets/36972116/b4e9a1c6-9841-4234-adfd-4682516c476b)
![Screenshot (39)](https://github.com/bassamanator/Sovol-SV06-firmware/assets/36972116/6d56bd3f-f26c-4dc1-88f7-7dc224043c0f)

**Edit:**
I guess bc the print start macro is expecting alot of info to be passed from the slicer the start gcode errors out. I put something more basic together to hopefully get the ball rolling I could always use a little help though
`G90 
g28
G92 E0
M190 S60
BED_MESH_PROFILE LOAD=default
G1 X0 Y0 Z50 F2000
M109 S215
purge_line
G92 E0`
I will attach the entire gcode test. Is PA only available in SS and OS?
[elegoo_blackpla.zip](https://github.com/bassamanator/Sovol-SV06-firmware/files/15140439/elegoo_blackpla.zip)

**Comment by bassamanator** [comment]
[pa_pattern-0.2to0.45-0.001incr.gcode.txt](https://github.com/user-attachments/files/15941975/pa_pattern-0.2to0.45-0.001incr.gcode.txt)

- For sure with OSS klipper configuration
- Ellis pattern generator
- 0.2 to 0.45; 0.001 increment
- For the `SV06`, but should fit without issue on the `Plus`
- Use a filament where the extrusion multiplier is `1`
- Keep your finger on the emergency button in case of collision
- Rename file to `.gcode`
- 230C filament temp, 60C bed temp
